net-analyzer/sn0int/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>git@rxv.cc</email>
+        <name>kpcyrd</name>
+    </maintainer>
```

Comment:
You need a Gentoo dev or a project to maintain/commit it for you in the Gentoo tree. Usually this is the proxy-maint project, but looks like @steev might be interested in helping out.

Reply:
confirm, package can't have external maintainer without proxy-maint@ or someone with @gentoo.org address in metadata.